src/Microsoft.ML.StaticPipe/LpNormalizerStaticExtensions.cs

```diff
@@ -52,10 +52,10 @@ public Reconciler(LpNormalizingEstimatorBase.NormFunction normKind, bool ensureZ
 
         /// <include file='../Microsoft.ML.Transforms/doc.xml' path='doc/members/member[@name="LpNormalize"]/*'/>
         /// <param name="input">The column to apply to.</param>
-        /// <param name="normKind">Type of norm to use to normalize each sample.</param>
+        /// <param name="norm">Type of norm to use to normalize each sample.</param>
         /// <param name="subMean">Subtract mean from each value before normalizing.</param>
         public static Vector<float> LpNormalize(this Vector<float> input,
-            LpNormalizingEstimatorBase.NormFunction normKind = LpNormalizingEstimatorBase.Defaults.Norm,
-            bool subMean = LpNormalizingEstimatorBase.Defaults.LpEnsureZeroMean) => new OutPipelineColumn(input, normKind, subMean);
+            LpNormalizingEstimatorBase.NormFunction norm = LpNormalizingEstimatorBase.Defaults.Norm,
```

Comment:
>norm [](start = 52, length = 4)

`Norm` is not usually used as a stand-alone. Perhaps `normalization`?

Reply:
Maybe no, my impression is that we prefer verb in static APIs.

---
In reply to: [264866893](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264866893) [](ancestors = 264866893)

Reply:
Yeah, let's stick with norm.

---
In reply to: [264918622](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264918622) [](ancestors = 264918622,264866893)